### ISSUE TITLE
2.0 Beta1 - Infinite loading screen

### ISSUE THREAD

**Issue by killam42** [unknownskl/greenlight#407]
Hello, I seem to be getting an infinite loading screen on the connecting to the console page. 
It seems to happen on both my home network and away from my home network. 
Posting this here to see if I'm alone in experiencing this issue.

Edit: forgot to add this is only affecting Console Streaming, Xcloud streaming works better than ever

**Comment by unknownskl** [comment]
On which hardware and os does this happen? It seems to happen the most on M1 macs

**Comment by MisterAnderson91** [comment]
I am getting this with a pop-up error message (sorry I didn't copy it but it included something like "Error: WaitingForServer" at the end).

Accepting the message leaves the main window still open at the "Loading" screen with the spinning circle, with the "Disconnect" button doing nothing, and having to quit the app and try again.

It doesn't happen every time, sometimes it connects fine to the xbox and starts streaming, usually on the second try from my testing. This lines up with the official android app which also often fails the first connection and works with the second.

M1 Macbook, macOS 12.5

**Comment by killam42** [comment]
> On which hardware and os does this happen? It seems to happen the most on M1 macs

M1 Macbook Pro, MacOS 12.6

**Comment by Paulc19-93** [comment]
Just read this thread and I am having the same issue. The hardware I am using is my steamdeck but I get that error message every time. 
Can never get it to even remote play. 
Cloud gaming works perfect though.

**Comment by unknownskl** [comment]
Can you stream the xbox via the official xbox app? (mobile or pc doesn't matter) If that works, can you try to connect using the client after that?
Booting the xbox from standby requires some configuration options like standby mode and remote streaming enabled in the settings.

**Comment by killam42** [comment]
I'm able to connect through the ios apps and the windows app. But now when I try through the client I get this
Error in Promise (400)

**Comment by antonywalker** [comment]
Likewise: I have been unable to connect. I have an Intel iMac, however— a 2019 model.

**Comment by killam42** [comment]
Now I'm getting through but I'm getting a black screen now, and no sound

**Comment by bartvincke** [comment]
I assume my issue is the same as the tread starten, namely that the xbox can not started from standby mode by Greenlight. 
When I open Greenlight beta1 appimage on steamdeck it logs in and shows my xbox ( series x ):

"Standby"
"Start stream"

I click it 

"Loading"
"We are getting your stream ready..."

after a while:

"Connect etimedout 51.11.5.173:443: {}"

it appears to be an azure IP( with no site running on it, maybe api )
It then goes back to the previous screen but never gets connected

"loading"
"We are getting your stream ready..."

I eventually cancel


When I use the windows Xbox app to connect to this same xbox, it just connects just fine ( the new way without turning on the light on the box and no hdmi output )
When I connect greenlight then ( closed the windows app first ) it just connects ( also new way ).

It appears as the wakeup signal from greenlight is not received or not being sent.
as far as I know there is no blokkage on the network or steamdeck ( afaik )

**Comment by AikenTho** [comment]
Same issues here too (error in promise,sometimes black screen and infinite loading) with a steam deck, 
Tried restart xbox, messing with settings, nat open, remote open well on my phone,
I have already reinstalled more than once and tried both version (2.0 and 1.2)
Any fix so far? Thank you

**Comment by jk779** [comment]
I think I'm having the same issue here. Checked out tag beta2 and the FIRST time i connected with yarn dev, i got a working stream, but with poor framerate. so i tried to connect again.

since then, im not able to connect anymore with

```
Error in Promise: {"state":"Failed","detailedSessionState":0,"errorDetails":{"code":"XccsUnknownError","message":"Agent : -2146697194 :  : State WaitingForServerToRegister"}}
```

doesnt workt with the executable built iwth yarn build either. restarted xbox, no change.

**Comment by jk779** [comment]
I think I'm having the same issue here. Checked out tag beta2 and the FIRST time i connected with yarn dev, i got a working stream, but with poor framerate. so i tried to connect again.

since then, im not able to connect anymore with

```
Error in Promise: {
  "state": "Failed",
  "detailedSessionState": 0,
  "errorDetails": {
    "code": "XccsUnknownError",
    "message": "Agent : -2146697194 :  : State WaitingForServerToRegister"
  }
}
```

doesnt workt with the executable built iwth yarn build either. restarted xbox, no change and endless loop after the lextron message from above.

![Bildschirm­foto 2023-02-09 um 05 04 45](https://user-images.githubusercontent.com/1104528/217715349-9c063e99-d754-48fd-bab2-ecb4d48ca166.png)

**Comment by MisterAnderson91** [comment]
I believe this is fixed in v2.0.0-beta3 if you want to check. You can close this issue if it's working.

**Comment by unknownskl** [comment]
2.0.1 release should have fixed this problem. If the problem persists then please reopen the issue.
